src/server/cards/adventures/Disciple.js

```diff
@@ -0,0 +1,19 @@
+import Card from 'cards/Card';
+
+export default class Disciple extends Card {
+  static cost = new Card.Cost({ coin: 5 });
+  static types = new Set('Action', 'Traveller');
+  async onPlay(player) {
+    // RIP
+  }
+
+  canTriggerOn(event, player) {
+    return event.name === 'discard' && event.triggeringPlayer === player && event.card === this && player.playArea.includes(this);
+  }
+
+  async onTrigger(event, player) {
+    event.handledByPlayer.set(player, true);
+    player.moveCard(this, player.playArea, player.game.supplies.get('Disciple'));
```

Comment:
I think we should use the return to supply function we already have.
```js
  returnToSupply(card, from = this.hand) {
    this.moveCard(card, from, this.game.supplies.get(card.title).cards);
  }
```

But obviously we can't use Gain on the second part